scripts/scripts.js

```diff
@@ -526,6 +526,12 @@ async function loadPage(doc) {
   loadDelayed(doc);
 }
 
+if (window.location.hostname === 'www.hlx.live') {
+  const url = `https://www.aem.live${window.location.pathname}${window.location.search}${window.location.hash}`;
+  // eslint-disable-next-line no-console
+  console.log(`redirecting to ${url}`);
```

Comment:
- Suggestion: use `URL` to chang hostname
- Actually redirect(?)
```suggestion
  const url = new URL(window.location.href);
  url.hostname = 'www.aem.live';
  // eslint-disable-next-line no-console
  console.log(`redirecting to ${url}`);
  window.location.href = url.href;
```

Reply:
i was trying that initially, but then even for testing got into weirdness with port and protocol on localhost. i think if anything we should use `origin` probably.

Reply:
i was trying that initially, but then even for testing got into weirdness with port and protocol on localhost. i think if anything we should use origin probably.